Managers/FrameMan.cpp

```diff
@@ -1,2651 +1,1391 @@
+					}
+				}
+
+				// Adjust glow areas for the player screen's position on the final buffer
+				for (list<Box>::iterator bItr = screenRelativeGlowBoxes.begin(); bItr != screenRelativeGlowBoxes.end(); ++bItr) {
+					g_PostProcessMan.GetPostScreenGlowBoxesList()->push_back(*bItr);
+					// Adjust each added glow area for the player screen's position on the final buffer
+					g_PostProcessMan.GetPostScreenGlowBoxesList()->back().m_Corner += screenOffset;
+				}
+			}
 		}
-	}
 
+		// Clears the pixels that have been revealed from the unseen layers
+		g_SceneMan.ClearSeenPixels();
 
-//////////////////////////////////////////////////////////////////////////////////////////
-// Method:          Clear
-//////////////////////////////////////////////////////////////////////////////////////////
-// Description:     Clears all the member variables of this FrameMan, effectively
-//                  resetting the members of this abstraction level only.
-
-void FrameMan::Clear()
-{
-    m_ResX = 640;
-    m_ResY = 480;
-    m_BPP = 8;
-    m_NewResX = m_ResX;
-    m_NewResY = m_ResY;
-    m_pBackBuffer8 = 0;
-	m_DrawNetworkBackBuffer = false;
-	m_StoreNetworkBackBuffer = false;
-	m_pBackBuffer32 = 0;
-    m_pScreendumpBuffer = 0;
-    m_PaletteFile.Reset();
-    m_pPaletteDataFile = 0;
-    m_BlackColor = 245;
-    m_AlmostBlackColor = 245;
-    m_Fullscreen = false;
-    m_NxWindowed = 1;
-    m_NxFullscreen = 1;
-    m_NewNxFullscreen = 1;
-    m_PostProcessing = false;
-    m_PostPixelGlow = false;
-    m_pYellowGlow = 0;
-    m_YellowGlowHash = 0;
-    m_pRedGlow = 0;
-    m_RedGlowHash = 0;
-    m_pBlueGlow = 0;
-    m_BlueGlowHash = 0;
-    m_PostScreenEffects.clear();
-    m_HSplit = false;
-    m_VSplit = false;
-    m_HSplitOverride = false;
-    m_VSplitOverride = false;
-    m_pPlayerScreen = 0;
-    m_PlayerScreenWidth = 0;
-    m_PlayerScreenHeight = 0;
-    m_PPM = 0;
-    m_pFrameTimer = 0;
-    m_MSPFs.clear();
-    m_MSPFAverage = 0;
-    m_SimSpeed = 1.0;
-    m_pGUIScreen = 0;
-    m_pLargeFont = 0;
-    m_pSmallFont = 0;
-    m_ShowPerfStats = false;
-	m_CurrentPing = 0;
-
-	m_NetworkFrameCurrent = 0;
-	m_NetworkFrameReady = 1;
-
-	// Delete all allocated primitives and clear the list
-	ClearPrimitivesList();
-
-    for (int i = 0; i < MAXSCREENCOUNT; ++i)
-    {
-        m_ScreenText[i].clear();
-        m_TextDuration[i] = -1;
-        m_TextDurationTimer[i].Reset();
-        m_TextBlinking[i] = 0;
-        m_TextCentered[i] = false;
-        m_FlashScreenColor[i] = -1;
-        m_FlashedLastFrame[i] = false;
-        m_FlashTimer[i].Reset();
-
-		for (int f = 0; f < 2; f++)
-		{
-			m_pNetworkBackBufferIntermediate8[f][i] = 0;
-			m_pNetworkBackBufferFinal8[f][i] = 0;
-			m_pNetworkBackBufferIntermediateGUI8[f][i] = 0;
-			m_pNetworkBackBufferFinalGUI8[f][i] = 0;
+		if (!m_StoreNetworkBackBuffer) {
+			// Draw split screen lines
+			acquire_bitmap(m_pBackBuffer8);
```

Comment:
The use of acquire_bitmap here seems weird to me. I think it's multiplayer related, since the corresponding release_bitmap is happening in a multiplayer section. Can you try moving this line to right above the `if (m_DrawNetworkBackBuffer) {` check? I think it shouldn't affect the separator line drawing stuff.